src/services/shipment_service.js

```diff
@@ -0,0 +1,199 @@
+     * @param {string} id
+     * @param {string} format
+     * @returns {this}
+     */
+    static async convertLabelFormat(id, format) {
+      try {
+        const wrappedParams = { file_format: format };
+        const response = await easypostClient.get(`${this._url}/${id}/label`, wrappedParams);
+
+        return this.convertToEasyPostObject(response.body);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Regenerate rates of a shipment.
+     * @param {string} id
+     * @param {boolean} withCarbonOffset
+     * @returns {this}
+     */
+    static async regenerateRates(id, withCarbonOffset = false) {
+      try {
+        const wrappedParams = { carbon_offset: withCarbonOffset };
+        const response = await easypostClient.post(`${this._url}/${id}/rerate`, wrappedParams);
+
+        return this.convertToEasyPostObject(response.body);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Get the smartrates of a shipment.
+     * @param {string} id
+     * @returns {this}
+     */
+    static async getSmartRates(id) {
+      try {
+        const response = await easypostClient.get(`${this._url}/${id}/smartrate`);
+
+        return this.convertToEasyPostObject(response.body.result);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Insure a shipment.
+     * @param {string} id
+     * @param {number|string} amount
+     * @returns {this}
+     */
+    static async insure(id, amount) {
+      try {
+        const wrappedParams = { amount };
+        const response = await easypostClient.post(`${this._url}/${id}/insure`, wrappedParams);
+
+        return this.convertToEasyPostObject(response.body);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Generate a form for a shipment.
+     * @param {string} id
+     * @param {string} formType
+     * @param {object} formOptions
+     * @returns {this}
+     */
+    static async generateForm(id, formType, formOptions = {}) {
+      try {
+        const wrappedParams = {
+          form: {
+            ...formOptions,
+            type: formType,
+          },
+        };
+        const response = await easypostClient.post(`${this._url}/${id}/forms`, wrappedParams);
+
+        return this.convertToEasyPostObject(response.body);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Refund a shipment.
+     * @param {string} id
+     * @returns {this}
+     */
+    static async refund(id) {
+      try {
+        const response = await easypostClient.post(`${this._url}/${id}/refund`);
+
+        return this.convertToEasyPostObject(response.body);
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+
+    /**
+     * Get the lowest smartrate of this shipment.
+     * @param {string} id
+     * @param {number} delivery_days
+     * @param {string} delivery_accuracy
```

Comment:
```suggestion
     * @param {number} deliveryDays
     * @param {string} deliveryAccuracy
```
These should be in camelCase